### ISSUE TITLE
duration calculation

### ISSUE THREAD

**Issue by oupala** [mudassir0909/jsonresume-theme-elegant#53]
How is calculated a duration between a startDate and an endDate?

When I use this json:

```json
"startDate": "1999-09-01",
"endDate": "2000-03-01",
```
I get the following result in my resume:

> Sep, 1999 - Mar, 2000 5 months

It should be 6 or 7 months, depending on how the duration is calculated, but not 5 months !

**Comment by mudassir0909** [comment]
It just uses [momentjs](http://momentjs.com/) 

```js
duration_in_milliseconds = moment(endDate) - moment(startDate)
duration = moment.duration(duration_in_milliseconds)
console.log(duration.years()); //=> 0
console.log(duration.months()); //=> 5
console.log(duration.days()); //=> 29
```

this is weird!

**Comment by oupala** [comment]
When using this version of moment : https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.2.1/moment.min.js

```js
var startDate = "1999-09-01";
var endDate = "2000-03-01";

duration_in_milliseconds = moment(endDate) - moment(startDate)
duration = moment.duration(duration_in_milliseconds)
console.log(duration.years()); //=> 0
console.log(duration.months()); //=> 6
console.log(duration.days()); //=> 2
```

**Comment by oupala** [comment]
Seen here: http://momentjs.com/docs/#/durations/

> Durations do not have a defined beginning and end date. They are contextless.
>
> A duration is conceptually more similar to '2 hours' than to 'between 2 and 4 pm today'. As such, they are not a good solution to converting between units that depend on context.
>
> For example, a year can be defined as 366 days, 365 days, 365.25 days, 12 months, or 52 weeks. Trying to convert years to days makes no sense without context. It is much better to use `moment#diff` for calculating days or years between two moments than to use `Durations`.

**Comment by oupala** [comment]
Here is a working way of using moment:

```js
var startDate = moment("1999-09-01");
var endDate = moment("2000-03-01");

console.log("years: " + endDate.diff(startDate, 'years')); //=> 0
console.log("months: " + endDate.diff(startDate, 'months')); //=> 6
console.log("days: " + endDate.diff(startDate, 'days')); //=> 182
```

**Comment by mudassir0909** [comment]
Aah, I'll push a fix this weekend, Thanks for pointing it out

**Comment by mudassir0909** [comment]
Fixed in 4d22a7046578ef97b31b7d70375d85857d0125b6
